.github/workflows/ci.yaml

```diff
@@ -1,45 +1,45 @@
-# yamllint disable rule:line-length
-# yamllint disable rule:braces
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow
 
 name: CI
 
 on:
-  pull_request:
+  pull_request: null
```

Comment:
This is the YAML default value. Strange!

Reply:
[The docs never set the default value](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions). I'd like to skip it here to because it is not an error, and it is confusing.

Reply:
I see what we have here in this PR.
You have no devops guy, and I do only devops.

Reply:
> [The docs never set the default value](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions).

GitHub docs is for people who don't have 101 hours to fiddle with YAML and bytes in general. Who have to deliver!